ci/jenkins/jobs/macros.yaml

```diff
@@ -223,3 +223,23 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-vm.sh  --registry ${DOCKER_REGISTRY} --kubeconfig /var/lib/jenkins/.kube/config
+
+- builder:
+    name: builder-rancher-e2e
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          chmod +x ./ci/jenkins/test-rancher.sh
+          ./ci/jenkins/test-rancher.sh --registry ${DOCKER_REGISTRY} --testcase e2e --cluster-name rancher-test
```

Comment:
1.There doesn't seem to be any testbed type parameter included in this code. 
2.As for the hardcoded cluster-name, it may not be the best approach as the hardcoded name lacks sufficient information, which could cause version confusion when upgrading the cluster. It might be more practical to use the Kubernetes version and testbed type to indicate the name of the Rancher cluster. 

If time is a constraint, we could consider addressing the first issue in this PR and leave the second one for another task.